xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -1404,10 +1404,10 @@ namespace PVR
       if (channelGroup)
       {
         // try to start playback of first channel in this group
-        std::vector<PVRChannelGroupMember> groupMembers(channelGroup->GetMembers());
+        const std::vector<std::shared_ptr<PVRChannelGroupMember>> groupMembers(channelGroup->GetMembers());
```

Comment:
Nowadays, I prefer default ctor + assignment over explicit ctor call. Reads more natural and modern compilers are smart enough to optimize this.

Reply:
I actually prefer this. too. Largely went with style that was already there. Happy to change.